contracts/prize-pool/PrizePool.sol

```diff
@@ -1050,6 +1051,15 @@ abstract contract PrizePool is PrizePoolInterface, OwnableUpgradeable, Reentranc
       compLike.delegate(to);
     }
   }
+  
+  /// @notice Required for ERC721 safe token transfers from smart contracts.
+  /// @param operator The address that acts on behalf of the owner
+  /// @param from The current owner of the NFT
+  /// @param tokenId The NFT to transfer
+  /// @param data Additional data with no specified format, sent in call to `_to`.
+  function onERC721Received(address operator, address from, uint256 tokenId, bytes calldata data) external override returns (bytes4){
+    return this.onERC721Received.selector;
```

Comment:
They recommend to use `IERC721.onERC721Received.selector` in the doc but I suppose it also works with `this`, maybe a bit less explicit though.
https://docs.openzeppelin.com/contracts/3.x/api/token/erc721#IERC721Receiver-onERC721Received-address-address-uint256-bytes-

Reply:
Updated to `IERC721ReceiverUpgradeable.onERC721Received.selector`